### PR TITLE
Fix/static analysis tools warnings

### PR DESCRIPTION
**Pull request by trombomart**

### CODE REVIEW

File: src/main/java/nl/tudelft/thefirstorder/service/util/PDFExportUtil.java [hunk 1]
```diff
@@ -49,7 +53,7 @@ public static Resource exportProjectToPDF(Project project) {
             addTitlePage(document, project);
             addContent(document, project);
             document.close();
-        } catch(DocumentException e) {
+        } catch (DocumentException e) {
```
Comment (rubenwiersma) [inline src/main/java/nl/tudelft/thefirstorder/service/util/PDFExportUtil.java]: Nice, added a space

**Comment by rubenwiersma** [comment]
Can't find anything wrong with this code.

File: build.gradle [hunk 2]
```diff
@@ -310,8 +310,8 @@ jacocoTestReport {
 
 tasks.withType(FindBugs) {
     reports {
-        xml.enabled = true
-        html.enabled = false
+        xml.enabled = false
```
Comment (hungnguyen94) [inline build.gradle]: This disables the xml which is printed by Travis. It's probably better to keep xml enabled if you commit, and use html when running locally.

**Comment by hungnguyen94** [comment]
I re-enabled the xml reporting for FindBugs. This is needed to output the report in Travis.

**Pull request merged**